src/PHPUnit/Controller/AbstractControllerTestCase.php

```diff
@@ -85,13 +81,10 @@ protected function tearDown(): void
      * If $traceError is true, appends exception details, if any.
      *
      * @deprecated (use LaminasContraint instead)
-     *
-     * @param string $message
-     * @return string
      */
-    protected function createFailureMessage($message)
+    protected function createFailureMessage(string $message): string
```

Comment:
Careful with this: while this is indeed a test package, adding type information to `protected` symbols is a BC break that leads to downstream crashes, in case of overrides.

Reply:
I removed risky type information of protected symbols.